goagen/gen_app/test_generator.go

```diff
@@ -59,9 +60,8 @@ func (g *Generator) generateResourceTest() error {
 		return nil
 	}
 	funcs := template.FuncMap{
-		"isSlice":       isSlice,
-		"initArguments": initArguments,
-		"tempvar":       tempvar,
+		"isSlice": isSlice,
+		"escape":  escape,
```

Comment:
`escape` should probably be a method on `Generator` this way you won't have to pass `$test` everywhere in the template (so `funcs` would be initialized with `"escape": g.escape`).